pinax/blog/context_processors.py

```diff
@@ -0,0 +1,7 @@
+from .conf import settings
+
+
+def scoped(request):
+    return {
+        "scoped_var": request.resolver_match.kwargs.get(settings.PINAX_BLOG_SCOPING_URL_VAR)
```

Comment:
`scoped_var` is a misleading name. The value of this key is really the `scoper` lookup value.